web/server/codechecker_server/auth/cc_ldap.py

```diff
@@ -139,22 +139,41 @@ def ldap_error_handler():
 def get_user_dn(con,
                 account_base_dn,
                 account_pattern,
-                scope=ldap.SCOPE_SUBTREE):
+                scope=ldap.SCOPE_SUBTREE,
+                user_dn_postfix_preference=None):
     """
     Search for the user dn based on the account pattern.
     Return the full user dn None if search failed.
+
+    user_dn_postfix_preference
```

Comment:
In this file we are using the following format to document a parameter:
https://github.com/Ericsson/codechecker/blob/d1604bbfdb1b0bcd9d1fc1034a157fe5a90ba672/web/server/codechecker_server/database/database.py#L452-L454
I think we should use this format other places too.

Reply:
ok, but in that case we should enforce it with the pydocstyle tool